Source/Core/Core/IPC_HLE/WII_IPC_HLE_Device_wfsi.cpp

```diff
@@ -0,0 +1,290 @@
+// Copyright 2016 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "Core/IPC_HLE/WII_IPC_HLE_Device_wfsi.h"
+
+#include <mbedtls/aes.h>
```

Comment:
Nope, each .cpp should be able to compile without depending on transitive includes.